docs/csharp/pattern-matching.md

```diff
@@ -1,268 +1,71 @@
 ---
-title: Pattern Matching - C# Guide
+title: Pattern Matching overview - C# Guide
 description: Learn about pattern matching expressions in C#
-ms.date: 04/10/2019
+ms.date: 04/26/2021
+ms.topic: overview
```

Comment:
@tdykstra 

Would you consider this an "overview"? It's an overview of the pattern matching features, which is more or less what the Contributor Guide says, but it's not in the correct node in the TOC yet. It's also more written for current C# developers than as part of an "onboarding' topic.

The other alternative is to leave it as the default.

Reply:
From what I recall of that discussion, "overview" was supposed to be limited to the one-overview-per-service MVC-compliant docs in the Azure doc set, and we never got approval to use it in our doc set. Whether we could use it or not was an open question that to my knowledge never got resolved, so I would stick with the default "conceptual".